client/src/styles/Student.css

```diff
@@ -59,12 +59,19 @@ button::-moz-focus-inner {
     text-align: center;
     font-size: 18px;
     display:inline-block;
-    float:right;
-    margin-right: 250px;
-    margin-top:5px;
+    margin-left: 10px;
+    margin-top: 13px;
     border: 1px solid #666;
 }
 
+#notBold{
+    font-weight:lighter;
+}
+
+.column{
```

Comment:
There are some other columns in the app that don't need to have a bigger font size, so I'll commit a change making this more specific.